docs/articles/expensify-classic/workspaces/reports/Scheduled-Submit.md

```diff
@@ -29,10 +29,15 @@ For individuals or employees: To enable Scheduled Submit on your individual work
 
 **Manually**: An open report will be created, and expenses will be added to it automatically. However, it's important to note that the report will not be submitted automatically; manual submission of reports will be required.This is a great option for automatically gathering all an employee’s expenses on a report for the employee’s convenience, but they will still need to review and submit the report.
 
+**Instant Submit**: Expenses are automatically added to a report in the Processing state, and all expenses will continue to accumulate on one report until it is Approved or Reimbursed. This removes the need to submit expenses, and Processing reports can either be Reimbursed right away (if Submit and Close is enabled), or Approved and then Reimbursed (if Submit and Approve is enabled) by a workspace admin.
```

Comment:
@stephanieelliott Shouldn't the `Scheduled Submit` option be named `Instantly` instead of `Instant Submit`? The reasoning is that the App displays as `Instantly` as shown below:

![Screenshot 2024-04-30 at 9 40 44 AM](https://github.com/Expensify/App/assets/3004659/361918c5-51c1-4d40-a2ad-0e8c42ee39d1)

Reply:
cc @stephanieelliott

Reply:
Discussed on [slack](https://expensify.slack.com/archives/C02QSAC6BJ8/p1714589267255109) since Steph is OOO and we're going with Instantly instead of Instant Submit.